utils/generate-bags/src/lib.rs

```diff
@@ -0,0 +1,277 @@
+// Copyright 2024, Horizen Labs, Inc.
+
```

Comment:
Shouldn't we add copyright header for Parity here as well  ?

Reply:
Added both as suggested by @la10736 (synced offline)